extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -68,6 +70,29 @@ amp-story:not([supports-landscape]) + amp-subscriptions-dialog.i-amphtml-story-s
   background: #EDEDED !important;
```

Comment:
We need this to be a flex container so that content will automatically move from left to right with RTL.

These are the rules for it. Do not copy and paste these. Please add them line by line and see how it places the content. in dev tools add `dir="rtl"` to the parent element, the content should automatically move to the other side.
display: flex;
justify-content: flex-end;  /* places content at end of box */
align-items: center; /* centers content vertically */

For this to work we need to remove `height`, `justify-content` and `align-items` from `i-amphtml-story-subscriptions-dialog-banner-button`.

Reply:
Good suggestion, done!